README.md

```diff
@@ -8,3 +8,6 @@
 | № | Имя Фамилия     | Любимый фильм                          |
 | - |:---------------:| --------------------------------------:|
 | 0 | Ава Катушка     | Три идиота                             |
+| 1 | Дима Зубков     | The Matrix                             |
+
+## Отправил 1 коммит без комментария, добавляю сюда закоменченную строку для того чтобы отправить 2 коммит
```

Comment:
Please, remove the comment from the file and change the language of the film to Russian.

Reply:
Good, thanks!